test/product_tests/AddToCart/AddToCart.test.js

```diff
@@ -18,26 +40,113 @@
     { quantity: 8, size: 'S', sku: '123457'},
   ]
 
-	render(<AddToCart sizeList={sizeListData}/>);
-
-  const sizeList = screen.getAllByRole("option");
-  const sizeOption = screen.getByRole("option", { name: 'XS'});
-  const quantityOption = screen.getByRole("option", { name: '7'});
-  const cartButton = screen.getByRole("button", { name: "Add To Cart" });
-
 	// Test 1
 	test("AddToCart Rendering, including size and quantity dropdown option, addToCart button", () => {
```

Comment:
I would maybe write more description test names so that I know what outcome you want for your test. Something like "item in cart increments when addToCart button is clicked" etc.

Reply:
Thanks for the advice. I'll add more details in the description next time.